omnigibson/prims/cloth_prim.py

```diff
@@ -130,6 +132,36 @@ def particle_positions(self, pos):
         p_local = (r.T @ (pos - t).T).T / s
         self.set_attribute(attr="points", val=array_to_vtarray(arr=p_local, element_type=Gf.Vec3f))
```

Comment:
fyi, sorry, this API is super slow. There's a much more optimal API you should call, `Vt.Vec3fArray.FromNumpy(input_array)`

Reply:
Did you call this new API in your refactoring PR? Can you modify this in your PR? I will like to keep this PR small for just the overlaid state if possible.